src/utils/nodes.ts

```diff
@@ -2,25 +2,26 @@ import { useIsFetching, useQuery } from '@tanstack/react-query'
 import { MinuteMs } from '../consts'
 import { getOperatorNodes } from '../getters'
 
-function getOperatorNodesForStreamQueryKey(streamId: string | undefined) {
-  return ['useOperatorNodesForStreamQuery', streamId || '']
+function getOperatorNodesForStreamQueryKey(streamId: string | undefined, chainId: number) {
```

Comment:
I'd prefer `chainId` to go first in all the chain-specific utility functions. It'd be
```ts
function foo(chainId: number, ...rest) {}
```

It then serves a role of a _directive_ for where to look for streams or nodes.

Low pressure.